cengage/react-magma
# feat(toggleButton): New ToggleButton & ToggleButtonGroup components

Issue: #904

## What I did
Adding
- ToggleButton component
- ToggleButtonGroup component

## Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/91160746/216425382-3c93d97c-6daf-4bae-b67c-cbf31c013c82.png)
![image](https://user-images.githubusercontent.com/91160746/216425398-dd257a43-563f-4f59-a7ee-4a86adce4267.png)
![image](https://user-images.githubusercontent.com/91160746/216425449-d5d69a8d-de99-4099-adb1-8d7d2afae4ef.png)


## Checklist 

## How to test
- Navigate to the ToggleButton component in storybook. Test all the controls
- Navigate to the ToggleButtonGroup component in storybook. Test all the controls. Confirm that `enforced` and `exclusive` work as expected
- Verify the documentation pages for both ToggleButton and ToggleButtonGroup
